Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py

```diff
@@ -131,7 +131,7 @@ def find_words_contain_tokens(positive_tokens, words_to_token_maps):
 def try_get_incident_field(field):
     value = ''
     incident = demisto.incident()
-    if 'CustomFields' in incident and field in incident['CustomFields']:
+    if 'CustomFields' in incident and incident['CustomFields'] is not None and field in incident['CustomFields']:
```

Comment:
Maybe you want to do:
if 'CustomFields' in incident and incident['CustomFields'] and field in incident['CustomFields']
So it won't be True if incident['CustomFields'] is []
Your choice

Reply:
you're right, but I think it's not too critical in this case,` incident['CustomFields']` is a dict or None

Reply:
and it cannot be an empty dict ?

Reply:
then `field in incident['CustomFields']` will be false (which in any case need to be checked)